nimbus.go

```diff
@@ -3,15 +3,54 @@
 package main
 
 import (
-	statusnim "github.com/status-im/status-nim"
-	"runtime"
+	"time"
+
+	"crypto/ecdsa"
+
+	"github.com/jroimartin/gocui"
+	nimbusbridge "github.com/status-im/status-protocol-go/bridge/nimbus"
+	whispertypes "github.com/status-im/status-protocol-go/transport/whisper/types"
 )
 
 func init() {
-	runtime.LockOSThread()
+	nimbusbridge.Init()
+}
+
+func startNimbus(privateKey *ecdsa.PrivateKey, listenAddr string, staging bool) error {
+	return nimbusbridge.StartNimbus(privateKey, listenAddr, staging)
+}
+
+func startPolling(g *gocui.Gui, pollFunc func(), delay time.Duration) chan bool {
```

Comment:
```suggestion
func startPolling(g *gocui.Gui, pollFunc func(), delay time.Duration, cancel <-chan struct{}) {
```
it's a bid odd that the control over the channel is handed to the caller.